test/data.jl

```diff
@@ -47,6 +47,10 @@
     ct = DataFrame(Tables.columns(t[1:3,:]))
     @test rv == rt
     @test cv == ct
+
+    # kwargs
+    sdata = georef((z=rand(100),))
+    @test length(EmpiricalHistogram(sdata, :z; nbins=10).weights) == 10
```

Comment:
Can you split this lines into two lines? The first computes the `h = EmpiricalHistogram(...)` and second checks the length.

Reply:
Yeah sure